deployment/proxy-deployment.yaml

```diff
@@ -0,0 +1,34 @@
+# https://kubernetes.io/docs/concepts/services-networking/ingress/#the-ingress-resource
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: proxy
+spec:
+  defaultBackend:
+    service:
+      name: ui
+      port:
+        number: 3000
+  rules:
```

Comment:
I think we need to add a rule so we can test FLEx in staging. Right now only requests that start with /api will go to the lexbox api, but we need all FLEx requests to come to lexbox. Can we have a rule for all the FLEx hosts detailed in the diagram?

Reply:
definitely on the backlog...I want to get our primary use case going for the app and then we can begin extending to those.  Thanks!